tripy/tripy/frontend/module/batchnorm.py

```diff
@@ -0,0 +1,116 @@
+#
+# SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from dataclasses import dataclass
+
+from tripy import export, utils
+from tripy.common import datatype
+from tripy.frontend.module.module import Module
+from tripy.frontend.module.parameter import DefaultParameter, Parameter
+
+@export.public_api(document_under="operations/modules")
+@dataclass
+@utils.constant_fields(["num_features"])
+class BatchNorm(Module):
```

Comment:
Can you add tests under `tests/integration`? You can refer to `tripy/tests/integration/test_layernorm.py` as an example

Reply:
I added a test against `torch.BatchNorm2D`. Let me know if you want tests `against` 1D and 3D ones as well, but I think their functionally the same as 2D one.